.github/pull_request_template.md

```diff
@@ -3,20 +3,12 @@
   It simply serves to provide a guide to allow a better review of pull requests.
 -->
 
-## What is in this PR?
-<!-- Please fulfill this section -->
-
-<!-- Please delete options that are not relevant. -->
-<!-- Add a link for each related issues (eg: [X] Bug fix #42 #21 #84 ) -->
-
-
-- [ ] Bug fix <!-- non-breaking change which fixes an issue -->
-- [ ] New feature <!-- non-breaking change which adds functionality) -->
-- [ ] Breaking change <!-- fix or feature that would cause existing functionality to not work as expected) -->
-
+<!--
+  IMPORTANT
+  Don't forget to add the corresponding label to your PR.
```

Comment:
(nitpicking) I think that the following is clearer: `on't forget to add the corresponding "changelog:xxx" label to your PR.`